src/RegistryCoordinator.sol

```diff
@@ -211,17 +233,16 @@ contract RegistryCoordinator is EIP712, Initializable, IRegistryCoordinator, ISo
 
         uint256 kickIndex = 0;
         for (uint256 i = 0; i < quorumNumbers.length; i++) {
-            uint8 quorumNumber = uint8(quorumNumbers[i]);
-            
-            OperatorSetParam memory operatorSetParams = _quorumParams[quorumNumber];
+            // reference: uint8 quorumNumber = uint8(quorumNumbers[i]);
+            OperatorSetParam memory operatorSetParams = _quorumParams[uint8(quorumNumbers[i])];
```

Comment:
Ah, is this avoiding "stack too deep"?

You may be able to avoid this change -- Yash discovered that we're not actually using `kickIndex` and has removed it in this PR: https://github.com/Layr-Labs/eigenlayer-middleware/pull/99

Reply:
hmm OK. yes this was the fix to my stack-too-deep issue

Reply:
I checked on this when chatting with Yash.

Previously (check master) - we didn't require the input length arrays to be the same size. Now, we do - so the index can just be `i`

Personally I like this better - it means a handful of the kick params could be empty, but it's less complicated than keeping 2 loop indices.

Reply:
ahhhh OK makes sense!
will delete this and revert the commented-out bit

Reply:
deleted here https://github.com/Layr-Labs/eigenlayer-middleware/pull/100/commits/9948e01a7701e1f9f4ea37e83ba3e9d4343bd852